### PR TITLE
Add net.lugsole.bible_gui

### PR DESCRIPTION
**Pull request by Lugsole**

### CODE REVIEW

File: net.lugsole.bible_gui.json [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    "sdk" : "org.gnome.Sdk",
+    "command" : "bible",
+    "finish-args" : [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--filesystem=xdg-run/dconf",
+        "--filesystem=~/.config/dconf:ro",
+        "--talk-name=ca.desrt.dconf",
+        "--env=DCONF_USER_CONFIG_DIR=.config/dconf",
```
Comment (bilelmoussaoui) [inline net.lugsole.bible_gui.json:14]:
```suggestion
```
Not needed anymore

**Comment by bilelmoussaoui** [comment]
bot, build net.lugsole.bible_gui

**Comment by Lugsole** [comment]
I'm new to this, whats next to getting this on the FlatHub app store?

File: net.lugsole.bible_gui.json [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+                }
+            ],
+            "buildsystem": "simple",
+            "build-commands": [
+            	"printenv",
+            	"pwd",
+            	"./autogen.sh",
+            	"./configure --bindir=/app/bin --libdir=/app/lib --prefix=/app/",
+                "make all",
+                "make install"
```
Comment (bilelmoussaoui) [inline net.lugsole.bible_gui.json:56]: Nitpick: split those into seperate lines.

File: net.lugsole.bible_gui.json [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+            ]
+        },
+        {
+            "name": "bible",
+            "builddir": true,
+            "buildsystem": "meson",
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/Lugsole/net.lugsole.bible_gui"
```
Comment (bilelmoussaoui) [inline net.lugsole.bible_gui.json:66]: This has to be a tagged release or a specific commit

**Comment by bilelmoussaoui** [comment]
The app-id doesn't look right to me, do you own `lugsole.net`?

**Comment by Lugsole** [comment]
Yea, I own `lugsole.net`

File: net.lugsole.bible_gui.json (hunk 3 above)
Comment (Lugsole) [reply net.lugsole.bible_gui.json]: Fixed...

File: net.lugsole.bible_gui.json (hunk 2 above)
Comment (bilelmoussaoui) [reply net.lugsole.bible_gui.json:56]: Not a nitpick anymore. This builds just fine without specifying the build-commands and using autotools (it's default one) as a build system

**Comment by nedrichards** [comment]
bot, build net.lugsole.bible_gui

**Comment by nedrichards** [comment]
Not blockers: Is there a way of adding more translations? I just see an empty expander in the preferences otherwise. Similar for pausing or stopping the playback?

Blocker: Switch to autotools rather than simple buildsystem for espeak.

Otherwise, lgtm.

**Comment by Lugsole** [comment]
bot, build net.lugsole.bible_gui

**Comment by Lugsole** [comment]
I think it kept failing then building espesk with multiple threads. I there a way to make a notee that it only works with one thread for some reason.

**Comment by Lugsole** [comment]
@nedrichards It does support other translations besides KJV, but other translations are complicated to package with the app in terms of copyright. In the app appdata.xml file it shares that it supports 3 file formats. he SQLite3 file format is the best so far. The files are in `~/.var/app/net.lugsole.bible_gui/data/`

translations

File: net.lugsole.bible_gui.json [hunk 4]
```diff
@@ -39,6 +39,7 @@
         },
         {
             "name": "espeak",
+	    "config-opts": ["--jobs=1"],
```
Comment (bilelmoussaoui) [inline net.lugsole.bible_gui.json:42]:
```suggestion           
           "no-parallel-make": true,
```

**Comment by Lugsole** [comment]
bot, build net.lugsole.bible_gui

File: net.lugsole.bible_gui.json (hunk 4 above)
Comment (Lugsole) [reply net.lugsole.bible_gui.json]: I did that.

**Comment by Lugsole** [comment]
What else is needed to get into the store?

**Pull request closed without merging**